apps/src/storage/dataBrowser/TableControls.jsx

```diff
@@ -38,7 +38,7 @@ class TableControls extends React.Component {
             <ConfirmDeleteButton
               body={msg.confirmClearTable()}
               buttonText={msg.clearTable()}
-              containerStyle={{width: 103, marginLeft: 10}}
```

Comment:
this hardcoded width was just essentially adding spacing between the buttons in the line which is now handled by the styles of the other buttons 

- [ ] double check LTR